README.md

```diff
@@ -22,5 +22,15 @@ Second, read the [Homebrew Troubleshooting Checklist](https://github.com/Homebre
 
 `brew help`, `man brew` or check the [Homebrew documentation](https://github.com/Homebrew/homebrew/tree/master/share/doc/homebrew#readme).
 
+## Contributing
+
+Have a look at the [Homebrew Formula Cookbook](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md).  In particular, your formula should pass `brew audit --strict <formula>` and `brew test <formula>`.  For example:
+
+```sh
+ln nodenv-my-plugin.rb "$(brew --prefix)/Library/Formulas"
```

Comment:
This line is unnecessary. You can just pass `brew audit` and `brew test` the path to the formula.